### PR TITLE
address TA feedback #3

### PR DESCRIPTION
**Pull request by ming0701**
Feedback:
https://github.com/UBC-MDS/olympic_medal_htest/issues/28
Question could be restated for clarity, something like: Is age associated with success at the Olympics?
Is the idea to compare age categories given all other features being equal? Why was 25 chosen as the cutoff?
What kind of visualization do you plan to make?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,22 +31,20 @@ The rendered final report is located [here](https://github.com/UBC-MDS/olympic_m
...
-1. Clone the GitHub repository and install the required [dependencies](#dependencies)
-2. Navigate into the root directory of the cloned repository on your local machine
-3. Run the `Makefile` file with the following command:
+1.  Clone the GitHub repository and install the required [dependencies](#dependencies)
+2.  Navigate into the root directory of the cloned repository on your local machine
+3.  Run the `Makefile` file with the following command:
 
+```{=html}
+<!-- -->
 ```
```
Comment (squisty) [inline README.md]: I'm not sure if the code block will render without another triple tick beneath the make statement

File: README.md [hunk 2]
```diff
@@ -29,22 +31,20 @@ The rendered final report is located [here](https://github.com/UBC-MDS/olympic_m
...
-```
+    make all
 
 The local repository can be reset to the initial cloned state by running:
 
-```
-make clean
-```
-
+    make clean
```
Comment (squisty) [inline README.md]: This one might also need to be surrounded by triple ticks, I can't preview the rendered version so these comments may or may not be relevant

**Review by squisty — COMMENTED** [review]
Everything looks good, only concerns I have I made comments about

**Comment by ming0701** [comment]
Not sure why, I didn't make the above changes you mentioned......are all these formatting automatically done by Rstudio?

**Comment by stevenleung2018** [comment]
I have reviewed the revisions in the narratives and they all look good.  But we need to clean up the formatting issues that @squisty pointed out.

If you suspect that RStudio is causing it, please avoid using the visual markdown editor function and make changes with text editing there.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
 
 ## Introduction
 
-For this project we will attempt to make a hypothesis test to answer the question: is the proportion of athletes younger than 25 that win a medal greater than the proportion of athletes of age 25 or older that win a medal? We chose this question and this topic since it is a pop culture subject for which we think strong domain is not really needed. It is important to note that the idea for this project is to be able to wrangle the data and test our hypothesis with the tools and techniques that we know how to use at the moment.
+For this project we will attempt to make a hypothesis test to answer the question: is the proportion of athletes younger than 25 that win a medal greater than the proportion of athletes of age 25 or older that win a medal? The age threshold of 25 years old was chosen as this is the median age of the athletes in the data set. We chose this question and this topic since it is a pop culture subject for which we think strong domain is not really needed. It is important to note that the idea for this project is to be able to wrangle the data and test our hypothesis with the tools and techniques that we know how to use at the moment.
```
Comment (stevenleung2018) [inline README.md:9]: This is great.  Thanks.

File: results/05_final_report.Rmd [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Under 25 vs Over 25 - Olympic Athletes
+title: Is age associated with success at the Olympics?
```
Comment (stevenleung2018) [inline results/05_final_report.Rmd:2]: This is good.  Thanks.

File: results/05_final_report.Rmd [hunk 5]
```diff
@@ -53,37 +52,37 @@ Here we attempt to perform a hypothesis test on the question, "Is the proportion
...
 
-**Limitations and assumptions**: 
+**Limitations and assumptions**:
 
-- The age threshold of 25 years old was chosen on intuition. If time was provided, a way to make this analysis more robust would be to make some research and talk to some domain experts to find out if 25 years old is a good threshold to set for this hypothesis test.
+-   The age threshold of 25 years old was chosen as this is the median age of the athletes in the data set. If time was provided, a way to make this analysis more robust would be to make some research and talk to some domain experts to find out if 25 years old is a good threshold to set for this hypothesis test.
```
Comment (stevenleung2018) [inline results/05_final_report.Rmd]: This is also good.  Thanks.

**Comment by ming0701** [comment]
Auto formatting done by Rstudio removed according to the above request.

**Pull request merged**